marti_status_msgs/package.xml

```diff
@@ -0,0 +1,22 @@
+<package format="2">
+  <name>marti_status_msgs</name>
+  <version>0.0.1</version>
```

Comment:
This needs to be the same version number as other packages in the repository (currently 0.2.0).

Reply:
Fixed